src/java/org/apache/cassandra/config/YamlConfigurationLoader.java

```diff
@@ -247,7 +296,8 @@ public Property getProperty(Class<? extends Object> type, String name)
                     @Override
                     public void set(Object o, Object o1) throws Exception
```

Comment:
nit: Maybe `o1` could be like `oldValue`?

Reply:
I suggest we leave it alone to be consistent with 4.0. 
I have to do one change there regarding duplication of warnings, there I can change it in both branches. I will put a note to myself. CASSANDRA-17160